### DISCUSSION TITLE
Complete beginner question

### DISCUSSION THREAD

**Discussion by benfrain** [nvim-orgmode/orgmode#140] [Q&A]
I’m wondering whether org mode would help in my scenario. 

I’m writing a book and it would be useful to have an outline on one side. 

Typically I have a *.md file for each chapter so seeing each file/chapter listed on one side, with a headline view for each section that i can move around, and have the section moved in the corresponding file would be fantastic. 

Is that the kind of thing org mode can be used for?

sorry, did say it was a beginner question!

**Comment by kristijanhusak** [comment]
I don't have any experience writing a book in any format, so I cannot help too much on this field.
Simple google search shows that people do this with help of orgmode exports and other similar tools.
Exports here are supported (via emacs cli), so you should be able to get similar experience.

**Reply by benfrain** [reply]
Thanks for the reply. I’m not worried about expert. It was more whether this gives any additional kind of organisation abilities for long form documents. So a side bar that shows headings only (for example), and deleting/moving that heading in the sidebar, would delete/move the entire section in the actual document.

**Reply by kristijanhusak** [reply]
There's no sidebar, but you can move sections into different documents with refiling (https://github.com/kristijanhusak/orgmode.nvim/blob/master/DOCS.md#org_refile). This basically prompts you to move specific section to a different file @ different section.

**Reply by benfrain** [reply]
Thanks, I’ll take a look
